### PR TITLE
Add nv4x fwupd verification documentation

### PR DESCRIPTION
**Pull request by mkopec**
Signed-off-by: Michał Kopeć <michal.kopec@3mdeb.com>

### CODE REVIEW

File: docs/variants/clevo_nv41/fwupd_usage.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+$ meson build
+$ ninja -C build
+$ sudo ninja -C build install
+```
+
+#### fwupd
+
+1. Install build dependencies:
+```bash
+$ sudo apt install -y abigail-tools bash-completion bubblewrap ca-certificates \
```
Comment (macpijan) [inline docs/variants/clevo_nv41/fwupd_usage.md:91]: @mkopec The setup `./contrib/setup` script I suggested was not enough for installing depednencies? The andavage is that it already supports multiple distributions.

File: docs/variants/clevo_nv41/fwupd_usage.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+$ sudo apt install ./libfwupd2_1.7.3+r68+gf3a5e4d1_amd64.deb
+$ sudo apt install ./libfwupd-dev_1.7.3+r68+gf3a5e4d1_amd64.deb
+$ sudo apt install ./libfwupdplugin4_1.7.3+r68+gf3a5e4d1_amd64.deb
+$ sudo apt install ./libfwupdplugin-dev_1.7.3+r68+gf3a5e4d1_amd64.deb
+$ sudo apt install ./libgusb2_0.3.5-1_amd64.deb
+$ sudo apt install ./libgusb-dev_0.3.5-1_amd64.deb
+```
+
+### Building from source
+
```
Comment (macpijan) [inline docs/variants/clevo_nv41/fwupd_usage.md:59]: @mkopec Let's add one sentence explaining that these steps should be only followed if [using binary pacakges](LINK THERE) does not work for you.

File: docs/variants/clevo_nv41/fwupd_usage.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+$ cd fwupd
+```
+
+3. Build and install fwupd:
+```bash
+$ sudo depmod
+$ meson build -Ddocs=none -Dplugin_flashrom=true
+$ ninja -C build
+$ sudo ninja -C build install
+```
```
Comment (macpijan) [inline docs/variants/clevo_nv41/fwupd_usage.md:120]: @mkopec We have build and installation procedures. Somehow we lost usage documentation (how to actually install update)?

Comment (mkopec) [reply docs/variants/clevo_nv41/fwupd_usage.md:120]: Looks like it got removed in your commit :) https://github.com/Dasharo/docs/pull/60/commits/28a12438273fea8ae31ee9fb4de4c26c60212ce1

File: docs/variants/clevo_nv41/fwupd_usage.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  libmm-glib-dev libqmi-glib-dev libmbim-glib-dev libpolkit-gobject-1-dev \
+  python3-gi-cairo python3-pygments python3-typogrify python3-toml \
+   python3-jinja2 python3-requests libsqlite3-dev systemd \
+  libsystemd-dev shared-mime-info texlive-fonts-recommended udev umockdev \
+  valac valgrind libtss2-dev shellcheck
+```
+
+2. Obtain source code:
+```bash
+$ git clone https://github.com/3mdeb/fwupd -b novacustom_nv41
```
Comment (macpijan) [inline docs/variants/clevo_nv41/fwupd_usage.md]: @mkopec I don't like branch, as it is a moving target. Please use `v1.7.3.1` tag from https://github.com/Dasharo/fwupd after confirming it is the same revision.

Comment (mkopec) [reply docs/variants/clevo_nv41/fwupd_usage.md]: Changed to `https://github.com/Dasharo/fwupd.git -b  v1.7.3.1`

File: docs/variants/clevo_nv41/fwupd_usage.md (hunk 3 above)
Comment (macpijan) [reply docs/variants/clevo_nv41/fwupd_usage.md:109]: Sorry for that. Please revert it back, please.

File: docs/variants/clevo_nv41/fwupd_usage.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+```bash
+$ sudo apt update
+$ sudo apt install -y build-essential pciutils libpci-dev libusb-1.0-0-dev \
+    cmake meson pkg-config libftdi1-dev debhelper git wget python3-markdown \
+    gcab
+```
+
+2. Obtain source code:
+```bash
+$ git clone https://github.com/flashrom/flashrom.git
```
Comment (macpijan) [inline docs/variants/clevo_nv41/fwupd_usage.md:78]: @mkopec Let's use tag `v1.2.0.1` from https://github.com/dasharo/flashrom (once you confirm that it's correct revision to use).

Comment (mkopec) [reply docs/variants/clevo_nv41/fwupd_usage.md]: It's correct. Updated

**Review by macpijan — APPROVED** [review]

**Pull request merged**